### ISSUE TITLE
Simple dubbo application log with warning by SpringExtensionFactory 

### ISSUE THREAD

**Issue by lonre** [apache/dubbo#6067]
simple dubbo application started with warning by SpringExtensionFactory, it is annoying.

some issues #2827 #4975

demo: https://github.com/lonre/dubbo-demo-warning
```txt
00:50:29.245 [main] INFO  o.s.c.s.ClassPathXmlApplicationContext - Refreshing org.springframework.context.support.ClassPathXmlApplicationContext@26ba2a48: startup date [Mon Apr 27 00:50:29 CST 2020]; root of context hierarchy
00:50:29.405 [main] INFO  o.s.b.f.xml.XmlBeanDefinitionReader - Loading XML bean definitions from class path resource [provider.xml]
00:50:29.794 [main] INFO  o.a.d.common.logger.LoggerFactory - using logger: org.apache.dubbo.common.logger.log4j.Log4jLoggerAdapter
00:50:29.900 [main] INFO  c.alibaba.spring.util.BeanRegistrar - The Infrastructure bean definition [Root bean: class [org.apache.dubbo.config.spring.beans.factory.annotation.DubboConfigAliasPostProcessor]; scope=; abstract=false; lazyInit=false; autowireMode=0; dependencyCheck=0; autowireCandidate=true; primary=false; factoryBeanName=null; factoryMethodName=null; initMethodName=null; destroyMethodName=nullwith name [dubboConfigAliasPostProcessor] has been registered.
00:50:30.142 [main] WARN  o.a.d.c.s.e.SpringExtensionFactory -  [DUBBO] No spring extension (bean) named:ssl, try to find an extension (bean) of type org.apache.dubbo.config.SslConfig, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.143 [main] WARN  o.a.d.c.s.e.SpringExtensionFactory -  [DUBBO] No spring extension (bean) named:monitor, try to find an extension (bean) of type org.apache.dubbo.config.MonitorConfig, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.143 [main] WARN  o.a.d.c.s.e.SpringExtensionFactory -  [DUBBO] No spring extension (bean) named:application, try to find an extension (bean) of type org.apache.dubbo.config.ApplicationConfig, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.143 [main] WARN  o.a.d.c.s.e.SpringExtensionFactory -  [DUBBO] No spring extension (bean) named:module, try to find an extension (bean) of type org.apache.dubbo.config.ModuleConfig, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.143 [main] WARN  o.a.d.c.s.e.SpringExtensionFactory -  [DUBBO] No spring extension (bean) named:metrics, try to find an extension (bean) of type org.apache.dubbo.config.MetricsConfig, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.667 [main] WARN  o.a.d.c.config.ConfigurationUtils -  [DUBBO] You specified the config center, but there's not even one single config item in it., dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.667 [main] WARN  o.a.d.c.config.ConfigurationUtils -  [DUBBO] You specified the config center, but there's not even one single config item in it., dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.743 [main] INFO  o.a.d.c.utils.ConfigValidationUtils -  [DUBBO] There's no valid monitor config found, if you want to open monitor statistics for Dubbo, please make sure your monitor is configured properly., dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.760 [main] INFO  o.a.d.c.bootstrap.DubboBootstrap -  [DUBBO] DubboBootstrap has been initialized!, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.760 [main] INFO  o.a.d.c.bootstrap.DubboBootstrap -  [DUBBO] DubboBootstrap is starting..., dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.820 [main] INFO  o.apache.dubbo.config.ServiceConfig -  [DUBBO] No valid ip found from environment, try to find valid host from DNS., dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.890 [main] WARN  o.a.d.c.s.e.SpringExtensionFactory -  [DUBBO] No spring extension (bean) named:logger, try to find an extension (bean) of type org.apache.dubbo.common.logger.Logger, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.919 [main] INFO  o.apache.dubbo.config.ServiceConfig -  [DUBBO] Export dubbo service org.apache.dubbo.demo.DemoService to local registry url : injvm://127.0.0.1/org.apache.dubbo.demo.DemoService?anyhost=true&application=hello-world-app&bind.ip=127.0.0.1&bind.port=20880&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&interface=org.apache.dubbo.demo.DemoService&methods=sayHello&pid=58135&release=2.7.6&side=provider&timestamp=1587919830785, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:30.920 [main] INFO  o.apache.dubbo.config.ServiceConfig -  [DUBBO] Register dubbo service org.apache.dubbo.demo.DemoService url dubbo://127.0.0.1:20880/org.apache.dubbo.demo.DemoService?anyhost=true&application=hello-world-app&bind.ip=127.0.0.1&bind.port=20880&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&interface=org.apache.dubbo.demo.DemoService&methods=sayHello&pid=58135&release=2.7.6&side=provider&timestamp=1587919830785 to registry registry://224.5.6.7:1234/org.apache.dubbo.registry.RegistryService?application=hello-world-app&dubbo=2.0.2&pid=58135&registry=multicast&release=2.7.6&timestamp=1587919830774, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:31.414 [main] INFO  org.apache.dubbo.qos.server.Server -  [DUBBO] qos-server bind localhost:22222, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:31.557 [main] INFO  o.a.d.r.transport.AbstractServer -  [DUBBO] Start NettyServer bind /0.0.0.0:20880, export /127.0.0.1:20880, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:31.605 [main] INFO  o.a.d.r.multicast.MulticastRegistry -  [DUBBO] Register: dubbo://127.0.0.1:20880/org.apache.dubbo.demo.DemoService?anyhost=true&application=hello-world-app&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&interface=org.apache.dubbo.demo.DemoService&methods=sayHello&pid=58135&release=2.7.6&side=provider&timestamp=1587919830785, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:31.606 [main] INFO  o.a.d.r.multicast.MulticastRegistry -  [DUBBO] Send multicast message: register dubbo://127.0.0.1:20880/org.apache.dubbo.demo.DemoService?anyhost=true&application=hello-world-app&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&interface=org.apache.dubbo.demo.DemoService&methods=sayHello&pid=58135&release=2.7.6&side=provider&timestamp=1587919830785 to /224.5.6.7:1234, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:31.607 [DubboMulticastRegistryReceiver] INFO  o.a.d.r.multicast.MulticastRegistry -  [DUBBO] Receive multicast message: register dubbo://127.0.0.1:20880/org.apache.dubbo.demo.DemoService?anyhost=true&application=hello-world-app&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&interface=org.apache.dubbo.demo.DemoService&methods=sayHello&pid=58135&release=2.7.6&side=provider&timestamp=1587919830785 from /192.168.0.2:1234, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:31.607 [main] INFO  o.a.d.r.multicast.MulticastRegistry -  [DUBBO] Subscribe: provider://127.0.0.1:20880/org.apache.dubbo.demo.DemoService?anyhost=true&application=hello-world-app&bind.ip=127.0.0.1&bind.port=20880&category=configurators&check=false&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&interface=org.apache.dubbo.demo.DemoService&methods=sayHello&pid=58135&release=2.7.6&side=provider&timestamp=1587919830785, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:31.608 [main] INFO  o.a.d.r.multicast.MulticastRegistry -  [DUBBO] Send multicast message: subscribe provider://127.0.0.1:20880/org.apache.dubbo.demo.DemoService?anyhost=true&application=hello-world-app&bind.ip=127.0.0.1&bind.port=20880&category=configurators&check=false&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&interface=org.apache.dubbo.demo.DemoService&methods=sayHello&pid=58135&release=2.7.6&side=provider&timestamp=1587919830785 to /224.5.6.7:1234, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:31.608 [DubboMulticastRegistryReceiver] INFO  o.a.d.r.multicast.MulticastRegistry -  [DUBBO] Receive multicast message: subscribe provider://127.0.0.1:20880/org.apache.dubbo.demo.DemoService?anyhost=true&application=hello-world-app&bind.ip=127.0.0.1&bind.port=20880&category=configurators&check=false&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&interface=org.apache.dubbo.demo.DemoService&methods=sayHello&pid=58135&release=2.7.6&side=provider&timestamp=1587919830785 from /192.168.0.2:1234, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:32.612 [main] WARN  o.a.d.r.multicast.MulticastRegistry -  [DUBBO] Ignore empty notify urls for subscribe url provider://127.0.0.1:20880/org.apache.dubbo.demo.DemoService?anyhost=true&application=hello-world-app&bind.ip=127.0.0.1&bind.port=20880&category=configurators&check=false&deprecated=false&dubbo=2.0.2&dynamic=true&generic=false&interface=org.apache.dubbo.demo.DemoService&methods=sayHello&pid=58135&release=2.7.6&side=provider&timestamp=1587919830785, dubbo version: 2.7.6, current host: 127.0.0.1
00:50:32.657 [main] INFO  o.a.d.m.DynamicConfigurationServiceNameMapping -  [DUBBO] Dubbo service[null] mapped to interface name[org.apache.dubbo.demo.DemoService]., dubbo version: 2.7.6, current host: 127.0.0.1
00:50:32.661 [main] INFO  o.a.d.c.bootstrap.DubboBootstrap -  [DUBBO] DubboBootstrap has started., dubbo version: 2.7.6, current host: 127.0.0.1
```

**Comment by lonre** [comment]
@chickenlj Why close this issue silently without any reponse for months?  😅

**Comment by AlbumenJ** [comment]
This issue was fix by #6343 .
